build_tools/travis/test_script.sh

```diff
@@ -34,10 +39,9 @@ run_tests() {
     export SKLEARN_SKIP_NETWORK_TESTS=1
 
     if [[ "$COVERAGE" == "true" ]]; then
-        nosetests -s --with-coverage --with-timer --timer-top-n 20 sklearn
-    else
-        nosetests -s --with-timer --timer-top-n 20 sklearn
+        TEST_CMD="$TEST_CMD --with-coverage"
```

Comment:
I'm not sure you mean to repeat `--with-coverage` here and in setting `TEST_CMD` above